spec/core/matchers/toHaveSpyInteractions.js

```diff
@@ -0,0 +1,57 @@
+describe('toHaveSpyInteractions', () => {
+  let spyObj;
+  beforeEach(() => {
+    spyObj = jasmineUnderTest.createSpyObj('NewClass', ['spyA', 'spyB']);
+    spyObj.otherMethod = function() {};
+  });
+
+  it('detects spy interactions', () => {
+    spyObj.spyA();
+    expect(spyObj).toHaveSpyInteractions();
```

Comment:
This (and other lines like it) tests the functionality of the Jasmine that's hosting the test run, not the Jasmine under test. See <https://github.com/jasmine/jasmine/blob/main/.github/CONTRIBUTING.md#self-testing>, <https://jasmine.github.io/pages/faq.html#self-test>, and neighboring files like toHaveBeenCalledSpec.js.

You might not be able to create a spy object via the obvious approach of calling `jasmineUnderTest.createSpyObj` -- I don't think we currently expose that method. In that case you can use `jasmineUnderTest.getEnv().createSpyObj` instead.

Reply:
Done. Thanks.